applications/sdp/mspi/Kconfig

```
@@ -2,4 +2,13 @@ config SDP_MSPI_MAX_RESPONSE_SIZE
 	int "Max size of response that can be sent to APP core. In bytes."
 	default 128
 
+config SDP_MSPI_IPC_NO_COPY
+	bool "IPC no copy mode"
+	default y
+	depends on SOC_NRF54L15
+	help
+	  If y Data is passed through IPC by reference,
+	  this requires both cores to be able to access each others memory spaces.
+	  If n Data is passed through IPC by copy.
```

Nit:
```suggestion
	  If enabled data is passed through IPC by reference,
	  this requires both cores to be able to access each others memory spaces.
	  If disabled data is passed through IPC by copy.
```
Same goes for FLPR side.